modules/sdk-core/src/bitgo/utils/opengpgUtils.ts

```diff
@@ -32,6 +32,33 @@ export async function getBitgoGpgPubKey(bitgo: BitGoBase): Promise<Key> {
   return await readKey({ armoredKey: bitgoPublicKeyStr });
 }
 
+/**
+ * Verify an Eddsa KeyShare Proof.
+ *
+ * @param senderPubKey public key of the sender of the privateShareProof
+ * @param privateShareProof u value proof
+ * @param uValue u value from an Eddsa keyshare
+ * @return {boolean} whether uValue proof actually was signed by sender as part of their subkeys
+ */
+export async function verifyShareProof(
+  senderPubKey: string,
+  privateShareProof: string,
+  uValue: string
+): Promise<boolean> {
+  const decodedProof = await pgp.readKey({ armoredKey: privateShareProof });
+  const otherOvcGpgKey = await pgp.readKey({ armoredKey: senderPubKey });
```

Comment:
this is the SDK, which means this can be used in other contexts (not just in OVC, but in hot wallets).   so remove the reference to "ovc" and use something akin to user/backup/bitgo since this will need to be used in hot wallets as well